File: api/examples/c#/PROXY_SERVER.cs
using System;

[Serializable()]
public partial class PROXY_SERVER
{
    public long ID = 0L;
    public string ProxyName = string.Empty;
    public COUNTRY Country = new COUNTRY();
    public string IP = string.Empty;
    public int Port = 0;
    public int ProxyType = 0;
    public int Flags = 0;
}

File: api/examples/vb.net/PROXY_SERVER.vb
<Serializable()>
Public Class PROXY_SERVER
    Public ID As Long = 0
    Public ProxyName As String = String.Empty
    Public Country As New COUNTRY
    Public IP As String = String.Empty
    Public Port As Integer = 0
    Public ProxyType As Integer = 0
    Public Flags As Integer = 0
End Class
